bitnami/airflow/templates/deployment-web.yaml

```
@@ -185,6 +185,32 @@ spec:
               {{- else }}
               value: "no"
               {{- end }}
+              {{- if .Values.ldap.enabled }}
+            - name: AIRFLOW_LDAP_ENABLE
+              value: yes
+            - name: AIRFLOW_LDAP_URI
+              value: {{ .Values.ldap.uri }}
+            - name: AIRFLOW_LDAP_SEARCH
+              value: {{ .Values.ldap.base }}
+            - name: AIRFLOW_LDAP_BIND_USER
+              value: {{ .Values.ldap.binddn }}
+            - name: AIRFLOW_LDAP_BIND_PASSWORD
+              value: {{ .Values.ldap.bindpw }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "airflow.ldapSecretName" . }}
+                  key: bind-password
+            - name: AIRFLOW_LDAP_UID_FIELD
+              value: {{ .Values.ldap.uidField }}
+              {{- if .Values.ldap.tls.enabled }}
+            - name: AIRFLOW_LDAP_USE_TLS
+              value: {{ .Values.ldap.tls.enabled }}
```

When `Values.ldap.tls.enabled` is true, I think it is safe to assume this will always be true, i.e.:

```suggestion
              value: "yes"
```